tldw_Server_API/app/core/Collections/reading_import_jobs.py

```
@@ -0,0 +1,181 @@
+    if not file_token:
+        raise ReadingImportJobError("reading_import_missing_file", retryable=False)
+    source = (payload.get("source") or "auto").strip().lower()
+    merge_tags = _coerce_bool(payload.get("merge_tags"), True)
+    filename = payload.get("filename")
+
+    import_path = resolve_reading_import_file(user_id, str(file_token))
+    raw_bytes = b""
+    try:
+        raw_bytes = import_path.read_bytes()
+        if not raw_bytes:
+            raise ReadingImportJobError("reading_import_empty", retryable=False)
```

Ruff (0.14.13)

Abstract `raise` to an inner function

(TRY301)